tests/testthat-1.R

```diff
@@ -1,4 +1,4 @@
 library(testthat)
 library(healthcareai)
 Sys.setenv("R_TESTS" = "")
-test_check("healthcareai", filter = "^[(a-m)|(A-M)]")
+test_check("healthcareai", filter = "^[(a-i)|(A-I)]")
```

Comment:
Why do we now have 3 testthat files?

Reply:
@mmastand This is to avoid the Travis time out errors. If travis doesn't get anything back from R in ten minutes, it fails the build. When one of these test files completes, it sends an update to Travis.